indigo_app/templates/document/_comparison.html

```diff
@@ -0,0 +1,32 @@
+<div class="document-comparison-view d-none">
+  <header class="comparison-header">
+    <div class="title">
+
+      <script id="comparison-header-template" type="text/x-handlebars-template">
+      {% verbatim %}
+        <h5>Differences between {{ document.expression_date }} (current content of this expression) and
+          <a href="/documents{{ comparison_document.id }}">{{ comparison_document.expression_date }}</a>
+        </h5>
+      {% endverbatim %}
+      </script>
+
+      <p>
+        Number of changes:
```

Comment:
UX: Prefer short statements, rather than key-value pairs, they're easier to read. eg "5 changes"